packages/zosfiles/__tests__/__system__/cli/delete/ds/cli.files.delete.ds.system.test.ts

```diff
@@ -111,6 +111,22 @@ describe("Delete Data Set", () => {
             expect(response.status).toBe(0);
             expect(response.stdout.toString()).toMatchSnapshot();
         });
+
+        it("should delete a partitioned data set member", async () => {
+            let response = runCliScript(__dirname + "/__scripts__/command/command_create_data_set.sh",
+                TEST_ENVIRONMENT, [dsname]);
+            const dsnameWithMember = dsname + "(TESTMEM)";
+            const fileLocation = __dirname + "/__scripts__/command/file.txt";
+            response = runCliScript(__dirname + "/__scripts__/command/command_create_data_set_member.sh",
+                TEST_ENVIRONMENT, [dsnameWithMember, fileLocation]);
+            response = runCliScript(__dirname + "/__scripts__/command/command_delete_data_set.sh",
+                TEST_ENVIRONMENT, [dsnameWithMember, "--for-sure", "--rfj"]);
+            expect(response.stderr.toString()).toBe("");
+            expect(response.status).toBe(0);
+            expect(response.stdout.toString()).toMatchSnapshot();
```

Comment:
We should try not to rely on snapshot testing for command output when possible.
No change required!